chia/full_node/weight_proof.py

```diff
@@ -527,7 +525,14 @@ def handle_block_vdfs(self, curr: HeaderBlock, blocks: Dict[bytes32, BlockRecord
             assert curr.reward_chain_block.challenge_chain_sp_vdf
             cc_sp_vdf_info = curr.reward_chain_block.challenge_chain_sp_vdf
             if not curr.challenge_chain_sp_proof.normalized_to_identity:
-                (_, _, _, _, cc_vdf_iters, _,) = get_signage_point_vdf_info(
+                (
+                    _,
+                    _,
+                    _,
+                    _,
+                    cc_vdf_iters,
+                    _,
```

Comment:
I'm not sure this is an improvement :/
maybe we need to remove the parentheses

Reply:
I would guess it's the trailing `,`.  Or maybe now regardless if the _whole_ thing doesn't fit on one line it does this.  I would also blame the ickiness here entirely on returning a tuple.